ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/modifier/Background.kt

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Detekt was mad about this line when I was in the file… I kinda want to revert this